test/integration/replicaset/test_replicaset.py

```diff
@@ -236,7 +237,6 @@ def test_status_custom_app(tt_cmd, tmpdir_with_cfg, flag):
 • .*
   Failover: unknown
   Master:   single
-    • test_custom_app  rw
```

Comment:
Is this affected by current changes?

Reply:
Yes, output is different. I decided to remove 1 line, rather than rewriting it.